ingestion-edge/requirements.txt

```diff
@@ -1,6 +1,5 @@
-blinker==1.4
-dockerflow==2018.4.0
-Flask==1.0.2
+aiohttp==3.4.4
+git+git://github.com/relud/python-dockerflow@master#egg=dockerflow
```

Comment:
temporary solution until dockerflow merges Sanic support and makes a release.

this should probably block merging this PR